src/scalper.ts

```diff
@@ -135,10 +132,10 @@ class Scalper {
           spotMarket,
         );
       } catch (err) {
-        console.log(this.symbol, 'Main Gamma Error', err.stack);
+        console.log(this.symbol, 'scalperOpenBook error', err.stack);
```

Comment:
Heplful to specify this error came from the gamma scalper rather than delta hedger